### PR TITLE
Image to add qe private tests

### PR DESCRIPTION
**Pull request by jhou1**

### CODE REVIEW

File: ci-operator/config/openshift/cucushift/OWNERS [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
```
Comment (liangxia) [inline ci-operator/config/openshift/cucushift/OWNERS:2]: I think we do not need this file manually.

File: ci-operator/config/openshift/cucushift/openshift-cucushift-master.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+build_root:
+  image_stream_tag:
+    name: verification-tests
```
Comment (liangxia) [inline ci-operator/config/openshift/cucushift/openshift-cucushift-master.yaml:3]: Is this name expected ?

Comment (jhou1) [reply ci-operator/config/openshift/cucushift/openshift-cucushift-master.yaml:3]: This is the build root present in this namespace, I have no idea which else to use

Comment (liangxia) [reply ci-operator/config/openshift/cucushift/openshift-cucushift-master.yaml:3]: ack

**Pull request merged**